src/domain/Game.java

```diff
@@ -82,6 +85,33 @@ private void enterPlayerNames(int amount) {
 			players.add(new Player(name));
 		}
 	}
+	
+	//method add currentPlayer to Game
+	private void currentPlayer(String currentPlayer) {
```

Comment:
Ik snap niet echt wat deze functie doet. Ik denk dat deze method weg kan en in de plaats moet er dan een `nextTurn` method komen die de volgende speler aan de beurt laat en die dus de `currentPlayer` maakt.